src/main/java/frc/robot/subsystems/ExampleSubsystem.java

```diff
@@ -35,9 +64,22 @@ public boolean exampleCondition() {
     return false;
   }
 
+  public void setSpeeds(double right, double left){
+    right = right * 0.5;
+    left = left * 0.5; //lowers speeds by scalar
```

Comment:
Maybe make this scalar a constant and then you can replace this magic number with a named variable. Plus you get a chance to use a nice, descriptive name.